app/src/main/AndroidManifest.xml

```diff
@@ -13,7 +14,8 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:networkSecurityConfig="@xml/network_security_config"
-        android:theme="@style/AppTheme">
+        android:theme="@style/AppTheme"
+        android:requestLegacyExternalStorage="true">
```

Comment:
I know during some of conversations we discussed that this external storage may not be needed. Have we tested that?

Reply:
@BottleRocket-Colin I'll check now